### PR TITLE
dependencies + readme tweak

### PR DESCRIPTION
**Pull request by ralphtheninja**
See https://github.com/Level/leveldown/issues/119

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -146,8 +147,7 @@ This leads to two alternative ways of managing a new LevelUP instance:
 levelup(location, options, function (err, db) {
   if (err) throw err
   db.get('foo', function (err, value) {
-    if (err) return console.log('foo does not exist')
-    console.log('got foo =', value)
+    if (!err) console.log('got foo =', value)
```
Comment (juliangruber) [inline README.md]: +1 to the others, this one and the one below were more complete before

Comment (ralphtheninja) [inline README.md]: Want me to revert this change or maybe make it more complete, just like the first example?
